rskj-core/src/main/java/co/rsk/rpc/exception/JsonRpcResponseLimitError.java

```diff
@@ -0,0 +1,12 @@
+package co.rsk.rpc.exception;
+
+public class JsonRpcResponseLimitError extends Error {
+    public static final int ERROR_CODE = -32011;
```

Comment:
do you think it'd make sense to define it in `JsonError` ?

Reply:
Updated how is done. Now the error code is stored in JsonRpcError class, and each exception returns a JsonRpcError with the corresponding code.